index.html

```diff
@@ -296,7 +296,7 @@ <h2>About me</h2>
   </div>
   <div>
   <form action="https://formspree.io" method="post">
-  <input type="text" id="name" name="name" placeholder="Full Name" maxlength="90" required>
+  <input type="text" id="name" name="name" placeholder="Full Name" maxlength="30" required>
   <input type="email" id="email" name="email" placeholder="Email Addres" required>
   <textarea name="textarea" placeholder="Enter text here" required></textarea>
```

Comment:
I would recommend adding a max-lentgth of `500` to the textarea as requested by the previous reviewer. Please note that this is a key requirement in this milestone.